content/5_Plat/Binary_Jump.mdx

```diff
@@ -129,6 +129,41 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```py
+import math
+
+
+n, q = map(int, input().split())
+
+
+def jmp(x: int, k: int) -> int:  # Get the k'th ancestor of x
+	for i in range(LOGMAX):
+		if k & 1 << i:
+			x = up[i][x]
+		if x == -2:
+			break
+	return x
+
+
+LOGMAX = math.ceil(math.log2(n)) + 1
```

Comment:
since this is calculated dynamically no need for all caps
also why the +1 here

Reply:
I removed +1